worker/modelcache/worker_test.go

```diff
@@ -278,13 +280,52 @@ func (s *WorkerSuite) TestAddApplication(c *gc.C) {
 	changes := s.captureApplicationEvents(c)
 	w := s.start(c)
 
-	app := s.Factory.MakeApplication(c, nil)
+	app := s.Factory.MakeApplication(c, &factory.ApplicationParams{Name: "app"})
 	s.State.StartSync()
-	_ = app
-	obtained := s.nextChange(c, changes)
-	_ = obtained
+
+	change := s.nextChange(c, changes)
+	obtained, ok := change.(cache.ApplicationChange)
+	c.Assert(ok, jc.IsTrue)
+	c.Check(obtained.Name, gc.Equals, app.Name())
```

Comment:
If we aren't asserting for the name "app", we can just exclude it from the factory MakeApplication args.

We don't actually care what the app is called, as long as the change has the same name as the app that we created with the factory.

Reply:
I added _something_ because if we compare empty strings, it isn't apparent whether one or the other was explicitly set, or neglected resulting in its zero-value.